ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx

```diff
@@ -263,17 +263,19 @@ export function VariableEditForm({
                   }}
                 />
               </Grid>
-              <Grid item xs={12}>
-                <ErrorBoundary FallbackComponent={() => <div>Error</div>}>
-                  <Stack direction={'row'} spacing={1} alignItems="center">
-                    <Typography variant="caption">Preview Values</Typography>
-                    <IconButton onClick={refreshPreview} size="small">
-                      <Refresh />
-                    </IconButton>
-                  </Stack>
-                  <VariableListPreview definition={previewSpec} />
-                </ErrorBoundary>
-              </Grid>
+              {state.listVariableFields.plugin.kind && (
+                <Grid item xs={12}>
+                  <ErrorBoundary FallbackComponent={() => <div>Error previewing values</div>} resetKeys={[previewSpec]}>
```

Comment:
A short comment inline might be helpful to someone in the future, not sure I would have 100% understood what was going on if I hadn’t read the PR description

Reply:
Oh wait I do see some comments further up now, ignore me if those cover things!